assets/l10n/app_en.arb

```diff
@@ -8,6 +8,6 @@
   "somethingWentWrong": "Something went wrong",
```

Comment:
I would suggest adding prefixes with the Screen when it us used, for instance, `errorScreenSomethingWentWrong` or `searchScreenSearchCharacter` so that in the future do not have a problems with the naming.